src/main/java/gregtech/integration/jei/recipe/GTRecipeWrapper.java

```diff
@@ -202,11 +202,21 @@ public void addIngredientTooltips(@NotNull Collection<String> tooltip, boolean n
                 double chance = entry.getChance() / 100.0;
                 double boost = entry.getChanceBoost() / 100.0;
                 if (logic != ChancedOutputLogic.NONE && logic != ChancedOutputLogic.OR) {
-                    tooltip.add(TooltipHelper.BLINKING_CYAN + I18n.format("gregtech.recipe.chance_logic",
-                            chance, boost, I18n.format(logic.getTranslationKey())));
+                    if (boost > 0) {
+                        tooltip.add(TooltipHelper.BLINKING_CYAN + I18n.format("gregtech.recipe.chance_logic",
+                                chance, boost, I18n.format(logic.getTranslationKey())));
+                    } else {
+                        tooltip.add(TooltipHelper.BLINKING_CYAN + I18n.format("gregtech.recipe.chance_logic_no_boost",
+                                chance, I18n.format(logic.getTranslationKey())));
+                    }
                 } else {
-                    tooltip.add(TooltipHelper.BLINKING_CYAN + I18n.format("gregtech.recipe.chance",
-                            chance, boost));
+                    if (boost > 0) {
+                        tooltip.add(TooltipHelper.BLINKING_CYAN + I18n.format("gregtech.recipe.chance",
+                                chance, boost));
+                    } else {
+                        tooltip.add(TooltipHelper.BLINKING_CYAN + I18n.format("gregtech.recipe.chance_no_boost",
+                                chance));
+                    }
```

Comment:
Why is this changing?

Reply:
This makes chances below 1% (as is common in fission fuel reprocessing) appear as something other than 0%.